coderd/database/dbfake/dbfake.go

```diff
@@ -4336,6 +4337,48 @@ func (q *FakeQuerier) InsertWorkspaceApp(_ context.Context, arg database.InsertW
 	return workspaceApp, nil
 }
 
+func (q *FakeQuerier) InsertWorkspaceAppStats(_ context.Context, arg database.InsertWorkspaceAppStatsParams) error {
+	err := validateDatabaseType(arg)
+	if err != nil {
+		return err
+	}
+
+	q.mutex.Lock()
+	defer q.mutex.Unlock()
+
+	nextID := int64(1)
+	if len(q.workspaceAppStats) > 0 {
+		nextID = q.workspaceAppStats[len(q.workspaceAppStats)-1].ID + 1
+	}
```

Comment:
Why not just keep track of last insert ID in `FakeQuerier` and increment on insert? It's unfortunate btw that we have to emulate this behaviour.

Reply:
Agreed it's unfortunate. The main reason I went this approach is we don't have a lot of precedent. I guess license inserts is the only one in fakedb using a last insert ID.

I'll make the change though since you asked about it. 👍🏻